utils/main.go

```diff
@@ -43,7 +43,11 @@ type sshExpiration struct {
 // CheckExpiredKey validates whether a key has expired. Keys with invalid expiration formats will result in an
 // error.
 func CheckExpiredKey(key string) error {
-	fields := strings.SplitN(key, " ", 4)
+  trimmedKey := strings.Trim(key, " ")
```

Comment:
i think the len(fields) check will catch this condition

Reply:
To clarity, would the empty string "    " be considered expired, or not expired?

When len(fields) checks the empty string, it returns a nil error, which seems to mean "not expired."

Reply:
great point, disregard my comment :)